### ISSUE TITLE
"Upscale 2x" button that actually uses upscale?

### ISSUE THREAD

**Issue by CaptainVarghoss** [mcmonkeyprojects/SwarmUI#122]
### Feature Idea

Some way to modify what process the 'upscale 2x' button does would be ideal, but simply having it use the upscale settings instead of whatever it's currently doing with the init image creativity.

Right off the bat, the label is misleading at best, being labeled as 'upscale 2x' one would expect it to use the upscale settings.
Second, the current method of upscaling does not use the upscaler model.
Third, as far as I can tell, I can't even modify the settings it uses, so the 0.4 'creativity' is destroying my images, even if the upscale was otherwise viable (it's not IMO)
Fourth, part of it's process is to completely regen the original image and then upscaling, wasting cycle time compared to simply turning on the upscale tab and reusing the seed.

In my mind, solving this could be done in a number of ways. 
At the minimum, a second button that simply uses the current upscaler settings (and maybe make the current one use the currently user set settings not whatever it uses)
At best, some way of making custom "turn these things on, generate with current settings, turn them back off" buttons would be amazing.

**Comment by CaptainVarghoss** [comment]
Submitted a pull request for this here - #308 

I added a setting for the current "Upscale 2x" button to change how much creativity is being used (0.4 is heavy handed IMO). Ultimately left the function and default value alone for backwards compatibility and those that like the way it currently works.

Added a second button called "Refine Image" that reuses the current images seed and enables the "Refine / Upscale" section and generates an image using the existing settings, then disables the section and resets the seed to random (-1).
